autoload/intero/process.vim

```diff
@@ -163,14 +164,103 @@ function! intero#process#restart() abort
 endfunction
 
 function! intero#process#restart_with_targets(...) abort
-    call intero#util#set_load_targets(a:000)
+    if a:0 == 0
+        let l:targets = s:prompt_for_targets()
+    else 
+        let l:targets = a:000
+    endif
+    call intero#targets#set_load_targets(l:targets)
```

Comment:
ugh, normally I have an editorconfig to prevent that. I've included one as part of the PR.